electron/renderer.js

```diff
@@ -57,13 +58,25 @@ class Renderer {
         try {
             this.redirectOutputToConsole();
             mocha.setup({
-                ui: 'tdd'
+                ui: 'tdd',
             });
 
-            let mochaInst = new Mocha();
+            // Format the reporter options
+            let reporterOptions;
+
+            // Parse string as an object
+            if (typeof this.options.reporterOptions === "string") {
```

Comment:
consider moving this logic beneath the mochaInst assignment?  would negate the need for a temp variable by just adding this property to the mochaInst object